src/modelio.cs

```diff
@@ -340,6 +345,11 @@ interface MDLLight
 		[Export ("lightType")]
 		MDLLightType LightType { get; set; }
 
+		// Added in iOS 10 SDK but it is supposed to be present in iOS 9.
+		[Mac (10,12)]
+		[Export ("colorSpace")]
+		string ColorSpace { get; set; }
```

Comment:
yes, it's unlikely to be a *user string* and, sometimes, constants do pointer comparison so we better (without any doc) not take a chance